aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -174,7 +174,7 @@ AT_ERROR("gesv: MAGMA library not found in "
     ipiv_array[i] = &ipiv_data[i * n];
   }
 
-  MAGMAQueue magma_queue(b.get_device());
+  MAGMAQueue magma_queue(b.device().index());
```

Comment:
Another task for future filing: `MAGMAQueue` should take Device directly and assert it's CUDA.